lib/lf/uscalfe/lagr_fe.h

```diff
@@ -710,6 +710,455 @@ class FeLagrangePoint : public ScalarReferenceFiniteElement<SCALAR> {
   unsigned degree_;
 };
 
+/**
+ * @headerfile lf/uscalfe/uscalfe.h
+ * @brief Quadratic Lagrangian finite element on a triangular reference element
+ *
+ * This is a specialization of ScalarReferenceFiniteElement.
+ * Refer to its documentation.
+ *
+ * The reference shape functions are combinations of the barycentric coordinate
+ * functions on the reference triangle.
+ *
+ * The first three shape functions are associated to the vertices,
+ * the other three shape functions to the edges of the triangle. There are
+ * no interior shape functions.
+ */
+template <typename SCALAR>
+class FeLagrangeO2Tria final : public ScalarReferenceFiniteElement<SCALAR> {
+ public:
+  FeLagrangeO2Tria(const FeLagrangeO2Tria&) = default;
+  FeLagrangeO2Tria(FeLagrangeO2Tria&&) noexcept = default;
+  FeLagrangeO2Tria& operator=(const FeLagrangeO2Tria&) = default;
+  FeLagrangeO2Tria& operator=(FeLagrangeO2Tria&&) noexcept = default;
+  FeLagrangeO2Tria() = default;
+  ~FeLagrangeO2Tria() override = default;
+
+  [[nodiscard]] base::RefEl RefEl() const override {
+    return base::RefEl::kTria();
+  }
+
+  [[nodiscard]] unsigned Degree() const override { return 2; }
+
+  /**
+   * @copydoc ScalarReferenceFiniteElement::NumRefShapeFunctions()
+   */
```

Comment:
I think you don't need to do a copydoc here. It should copy the documentation automatically because you override it.